lisp/mastodon-tl.el

```diff
@@ -1,4 +1,4 @@
-;;; mastodon-tl.el --- HTTP request/response functions for mastodon.el  -*- lexical-binding: t -*-
```

Comment:
a) all the files seem to use 3 semicolons, why change it in only one?
b) this seems unrelated to this PR.